skyportal/handlers/api/invitations.py

```diff
@@ -359,9 +380,15 @@ def patch(self, invitation_id):
         group_ids = data.get("groupIDs")
         stream_ids = data.get("streamIDs")
         role_id = data.get("role")
-        if group_ids is None and stream_ids is None and role_id is None:
+        user_expiration_date = data.get("userExpirationDate")
+        if (
+            group_ids is None
+            and stream_ids is None
+            and role_id is None
+            and user_expiration_date is None
+        ):
             return self.error(
-                "At least one of `groupIDs`, `streamIDs` or `role` is required."
+                "At least one of `groupIDs`, `streamIDs`, `role`, or `userExpirationDate` is required."
```

Comment:
Just double checking: can a user then be created with just an expiration date?  Is that desired?

Reply:
This is in the PATCH handler and not the POST. The assumption is that the required info is already there, and we're allowing users to edit just one of those fields if desired.

Reply:
Got it.